src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -6,6 +6,275 @@ cssPrefix: pf-l-flex
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+{{/l-flex}}
+<br/>
+<h3>
+  Row gap using <code>.pf-m-row-gap</code>
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-row-gap-sm"}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+{{/l-flex}}
+<br>
+<h3>
+  Column gap using <code>.pf-m-column-gap</code>
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-column-gap-lg"}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+{{/l-flex}}
+<br>
+<h3>
+  Gap inherits but can be overridden when nested - <code>.pf-l-gap-xl > .pf-l-gap-xs</code>
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-gap-xl"}}
+  {{#> l-flex l-flex--modifier="pf-m-gap-xs"}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+  {{/l-flex}}
+  {{#> l-flex}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+  {{/l-flex}}
+{{/l-flex}}
+<br>
+<h3>
+  REMOVE AFTER TESTING Nested flex and items
+</h3>
+{{#> l-flex l-flex--modifier="pf-m-gap-xs"}}
+  {{#> l-flex l-flex--modifier="pf-m-gap-xl"}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+  {{/l-flex}}
+
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+  {{#> l-flex-item}}
+    Flex item
+  {{/l-flex-item}}
+
+  {{#> l-flex l-flex--modifier="pf-m-gap-xs"}}
+    {{#> l-flex-item}}
+      Flex item
+    {{/l-flex-item}}
+  {{/l-flex}}
+{{/l-flex}}
+<br>
+
+<h3>
+  Gap with individually spaced items - <code>.pf-m-gap-xs > pf-m-spacer-{xs,sm,md,lg,xl,2xl,3xl}</code>.
```

Comment:
oops! Forgot to submit this comment in my previous review

This works if the spacer is larger than the gap, but if the gap is larger than the spacer, the item will have a negative margin, causing it to go out of bounds.

<img width="541" alt="Screenshot 2022-12-08 at 6 32 48 PM" src="https://user-images.githubusercontent.com/35148959/206596711-052b6c48-c938-47aa-a476-da386f809d8a.png">